src/lane_follow/include/LineDetect.h

```diff
@@ -0,0 +1,194 @@
+/*------------------------------------------------------------------------------------------*\
+Lane Detection
+
+General idea and some code modified from:
+chapter 7 of Computer Vision Programming using the OpenCV Library.
+by Robert Laganiere, Packt Publishing, 2011.
+
+ This program is free software; permission is hereby granted to use, copy, modify,
+and distribute this source code, or portions thereof, for any purpose, without fee,
+subject to the restriction that the copyright notice may not be removed
+or altered from any source or altered source distribution.
+The software is released on an as-is basis and without any warranties of any kind.
+In particular, the software is not guaranteed to be fault-tolerant or free from failure.
+The author disclaims all warranties with regard to this software, any use,
+and any consequent failure, is purely the responsibility of the user.
+ 
+Copyright (C) 2013 Jason Dorweiler, www.transistor.io
+\*------------------------------------------------------------------------------------------*/
+
+/*
+ * Created by: Raad Khan
+ * Created On: April 23, 2017
+ * Description:
+ * Usage:
+ * Subscribes to:
+ * Publishes to:
+ *
+ */
+
+#ifndef LINEF
+#define LINEF
+
+#include <ros/ros.h>
+#include <ros/package.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <ros/console.h>
+#include <sb_utils.h>
+#include <ros/time.h>
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include "opencv2/highgui/highgui.hpp"
+#include <opencv2/objdetect/objdetect.hpp>
+#include <iostream>
+#include <vector>
+#include <stdio.h>
+
+class LineFinder {
+
+public:
+
+    /**
+     * Constructor
+     */
+    RosVision(int argc, char** argv, std::string node_name);
+
+    RosVision();
+
+public:
+
+	// Default accumulator resolution is 1 pixel by 1 degree
+	// no gap, no mimimum length
+	LineFinder() : deltaRho(1), deltaTheta(PI/180), minVote(10), minLength(0.), maxGap(0.) {}

```

Comment:
Very clean, I like this. Just to be a more explicit, we should probably replace `0.` with `0.0`